src/main/java/com/maxmind/db/Reader.java

```diff
@@ -190,7 +198,61 @@ record = this.readNode(buffer, record, bit);
         return new DatabaseRecord<>(dataRecord, ipAddress, pl);
     }
 
-    private BufferHolder getBufferHolder() throws ClosedDatabaseException {
+    /**
+     * Creates a Networks iterator and skips aliased networks.
+     * Please note that a MaxMind DB may map IPv4 networks into several locations
+     * in an IPv6 database. This iterator will iterate over all of these locations
```

Comment:
I think this is wrong now. It will only iterate over the canonical location, not the aliases.